srcpkgs/Minder/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'com.github.phase1geo.Minder'
+pkgname=Minder
+version=1.0.8
+revision=1
+build_style=meson
+hostmakedepends="git pkg-config"
+makedepends="vala-devel glib-devel cairo-devel granite-devel gtk+3-devel libxml2-devel libgee08-devel"
+depends=""
+short_desc="Mind-mapping application designed for Pantheon"
+maintainer="Cameron Nemo <camerontnorman@gmail.com>"
+license="GPL-3.0-only"
```

Comment:
GPL-3.0-or-later

Reply:
huh looked like 3.0-only to me, how can I tell in the future?